KNOWN_ISSUES.md

```diff
@@ -9,15 +9,11 @@ Please report any other issue you may encounter by using the app help form.
 * Trashing, restoring and deleting content permanently may not work as expected
   yet, especially regarding shared folders and connectors.
   [details](https://trello.com/c/6jfO4hoB)
-* Changing only the case or encoding in a file or directory name, e.g. renaming
-  *my stuff* to *MY STUFF*, won't be synchronized on other devices.
-  [details](https://trello.com/c/Phc3lLEr)
 * Symlinks are not supported yet. Symlinks inside the synchronized directory
   will trigger errors and their content won't be synchronized.
 * Using a mount point as the synchronized folder can have unexpected
   consequences: in case it is not mounted when the client starts, the directory
-  will appear empty, the client will assume you deleted everything and will
-  update your Cozy accordingly, putting everything in the Cozy trash.
+  will appear missing and the client will ask you to choose another one.
```

Comment:
I believe we don't handle an empty sync dir correctly in the atom watcher yet:

| watcher | sync dir missing | sync dir empty |
| --- | --- | --- |
| atom | ok | **ko** |
| chokidar | ok | ok |

Reply:
What would happen in this case? Would we be emptying the folder again?